lib/rubygems/core_ext/kernel_warn.rb

```diff
@@ -5,21 +5,19 @@
 
   module Kernel
     rubygems_path = "#{__dir__}/" # Frames to be skipped start with this path.
-
```

Comment:
Please avoid unnecessary whitespace changes. Thanks!

Reply:
I think those were due to RuboCop rules, but we change them, it seems like a good opportunity for me to clean them, as the original author (7153a317c3e545658459d84aeafdad891d3a8a36) did not intend those extra lines.
Is it OK to keep that change?
I can revert that if needed, but IMHO it's more readable this way.

Reply:
Let's focus on the fix itself and leave style appeasing tweaks aside.

Reply:
Done